src/Modules/Backend/tests/Backend/Actions/UserAddTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace ForkCMS\Modules\Backend\tests\Backend\Actions;
+
+use ForkCMS\Modules\Backend\tests\BackendWebTestCase;
+
+final class UserAddTest extends BackendWebTestCase
+{
+    protected const TEST_URL = '/private/en/backend/user-add';
+
+    public function testPageLoads(): void
+    {
+        self::loginBackendUser();
+        self::assertPageLoadedCorrectly(
+            self::TEST_URL,
+            'Add | users | settings | Fork CMS | Fork CMS',
+            [
+                'Display name',
+                'E-mail',
+                'Super admin (grant access to everything)',
+                'Enable CMS access for this account.',
+                'Short date format',
+            ]
+        );
+        self::assertHasLink('Cancel', '/private/en/backend/user-index');
+    }
+
+    public function testEmptyFormShowsValidationErrors(): void
+    {
+        self::loadPage();
+        self::assertEmptyFormSubmission('user', 3);
+    }
+
+    public function testValidData(): void
```

Comment:
`testWithInvalidData` as you are testing with invalid data?

Reply:
Fixed in f9a393b8c7ae39260fdf9933aa0282408c2cac35